test/cpp/api/parallel.cpp

```diff
@@ -55,7 +55,7 @@ TEST_F(ParallelTest, DifferentiableGather_MultiCUDA) {
   ASSERT_EQ(output.device(), torch::Device(torch::kCUDA, 1));
 
   auto chunks = output.chunk(2);
-  ASSERT_TRUE(chunks[0].to({torch::kCUDA, 0}).allclose(a));
+  ASSERT_TRUE(chunks[0].to(torch::Device{torch::kCUDA, 0}).allclose(a));
```

Comment:
does this not work anymore?

Reply:
It doesn't. From the commit message:

New constructors for Tensor seem to cause some brace-enclosed
initializer lists to become ambiguous. I fixed the one case where this
happened; in the future, we may want to reduce the number of tensor
constructors.